From 043a3cd29115a97402d44a3ecb40cbd1bc8587c7 Mon Sep 17 00:00:00 2001
From: Vladimir Oltean <olteanv@gmail.com>
Date: Tue, 5 Jun 2018 01:11:12 +0300
Subject: [PATCH 08/28] Revert "extract_utils: Fix makefile generation issues"

This reverts commit c982836ca6292ac0dd1d2e326be22dc1131b2484.
This breaks extracting from src in "src:dst", such as from a disk image.
Will be addressed in the cleanup commit that follows.

Change-Id: Iff84a926f0c3bf908320b43ba40235e0a89db644
---
 build/tools/extract_utils.sh | 1 -
 1 file changed, 1 deletion(-)

diff --git a/build/tools/extract_utils.sh b/build/tools/extract_utils.sh
index 7de8e36a..4b919919 100644
--- a/build/tools/extract_utils.sh
+++ b/build/tools/extract_utils.sh
@@ -655,7 +655,6 @@ function parse_file_list() {
 
         # if line starts with a dash, it needs to be packaged
         if [[ "$SPEC" =~ ^- ]]; then
-            SPEC=$(echo "${SPEC}" | sed 's/[^"]*://')
             PRODUCT_PACKAGES_LIST+=("${SPEC#-}")
             PRODUCT_PACKAGES_HASHES+=("$HASH")
         else
-- 
2.17.1

